lttoolbox-language-module/configure.ac

```diff
@@ -2,21 +2,12 @@ AC_PREREQ(2.52)
 
 AC_INIT([Apertium {{languageName}}], [0.1.0], [{{email}}], [apertium-{{languageCode}}], [http://wiki.apertium.org/wiki/Apertium-{{languageCode}}])
 AM_INIT_AUTOMAKE
-
-AC_PROG_LN_S
```

Comment:
? Was this just removed because it's unclear why it's here?

Reply:
I've been removing that from all releases, because it's superfluous. We don't even use `ln -s` anywhere, but all platforms support it. By comparison, I don't remove the awk line because that's actually needed.